webpages/settings/data/addon-groups.js

```diff
@@ -92,4 +92,12 @@ export default [
     iframeShow: false,
     fullscreenShow: true,
   },
+  {
+    id: "bugfix",
+    name: "bugfix",
+    addonIds: [],
+    expanded: false,
+    iframeShow: false,
+    fullscreenShow: true,
+  },
```

Comment:
Adding a group here, without modifying index.js, won't do anything.

Reply:
I know, but now this `Bugfix` tag is just indicator - it don't have specific group in settings page etc. But in the future, when someone wants to change it, the basics will be ready!